nodeinstaller/node-installer.go

```diff
@@ -249,8 +249,27 @@ func patchContainerdConfig(runtimeHandler, basePath, configPath string, platform
 		return nil
 	}
 
+	// Backup the existing config.
+	if len(existingRaw) != 0 {
+		if err := os.WriteFile(fmt.Sprintf("%s.%d.bak", configPath, time.Now().Unix()), existingRaw, 0o666); err != nil {
```

Comment:
Could you please add a sentence to  https://docs.edgeless.systems/contrast/components/runtime#node-level-components that describes the process and the backup, and add a link to that page to https://docs.edgeless.systems/contrast/deployment#deploy-the-contrast-runtime?